extension/address/deleteAddresses.js

```diff
@@ -0,0 +1,47 @@
+const InternalError = require('./../common/Error/InternalError')
+const InvalidCallError = require('./../common/Error/InvalidCallError')
+const NotFoundError = require('./../common/Error/NotFoundError')
+
+/**
+ * @param {SDKContext} context
+ * @param {{ids: string[]}} input
+ * @return {Promise<>}
+ */
+module.exports = async (context, input) => {
+  if (!input.ids) {
```

Comment:
Is that even possible? Pipelines that are called with wrong parameters won't get executed anyway?